util/libcrypto.num

```diff
@@ -5739,3 +5739,13 @@ i2d_OSSL_AUTHORITY_ATTRIBUTE_ID_SYNTAX  ?	3_5_0	EXIST::FUNCTION:
 OSSL_AUTHORITY_ATTRIBUTE_ID_SYNTAX_free ?	3_5_0	EXIST::FUNCTION:
 OSSL_AUTHORITY_ATTRIBUTE_ID_SYNTAX_new  ?	3_5_0	EXIST::FUNCTION:
 OSSL_AUTHORITY_ATTRIBUTE_ID_SYNTAX_it   ?	3_5_0	EXIST::FUNCTION:
+d2i_OSSL_ROLE_SPEC_CERT_ID              ?	3_4_0	EXIST::FUNCTION:
+i2d_OSSL_ROLE_SPEC_CERT_ID              ?	3_4_0	EXIST::FUNCTION:
+OSSL_ROLE_SPEC_CERT_ID_free             ?	3_4_0	EXIST::FUNCTION:
+OSSL_ROLE_SPEC_CERT_ID_new              ?	3_4_0	EXIST::FUNCTION:
+OSSL_ROLE_SPEC_CERT_ID_it               ?	3_4_0	EXIST::FUNCTION:
+d2i_OSSL_ROLE_SPEC_CERT_ID_SYNTAX       ?	3_4_0	EXIST::FUNCTION:
+i2d_OSSL_ROLE_SPEC_CERT_ID_SYNTAX       ?	3_4_0	EXIST::FUNCTION:
+OSSL_ROLE_SPEC_CERT_ID_SYNTAX_free      ?	3_4_0	EXIST::FUNCTION:
+OSSL_ROLE_SPEC_CERT_ID_SYNTAX_new       ?	3_4_0	EXIST::FUNCTION:
+OSSL_ROLE_SPEC_CERT_ID_SYNTAX_it        ?	3_4_0	EXIST::FUNCTION:
```

Comment:
The version numbers are wrong here - they should be 3_5_0

Reply:
Fixed.